### PR TITLE
Support branch selection

### PR DESCRIPTION
**Pull request by yuta0801**
I made some changes to support to select branch.

### CODE REVIEW

**Comment by vinicius0026** [comment]
Hi @yuta0801!

Thanks for creating a PR to this module.

I don't actually use this module anymore. I'll merge your PR and release a new version, but I'm not able to test it in any way. I skimmed through your changes and everything seems ok.

If this module is important to you, I can add you as a collaborator if you want to.

**Comment by vinicius0026** [comment]
@Asuza Do you still use this module? If so, are you ok with the changes and with releasing a 0.2 version?

File: lib/getConfig.js [hunk 1]
```diff
@@ -24,8 +29,8 @@ var inquirer = require('inquirer'),
     }, {
         type: 'input',
         name: 'updateCmd',
-        message: 'Shell command to be issued to fetch the new version of the app. Usually git pull origin master',
-        default: 'git pull origin master'
+        message: 'Shell command to be issued to fetch the new version of the app. `{branch}` will replace by branch name.',
```
Comment (Asuza) [inline lib/getConfig.js]: The English here is a little weird. Should this say `'{branch}' will be replaced by branch name.`?

**Comment by Asuza** [comment]
@vinicius0026 I do. While I didn't test it, the changes seem pretty straight-forward and like a good improvement overall. I left one comment in the review, but otherwise it looks fine.

File: lib/getConfig.js (hunk 1 above)
Comment (yuta0801) [reply lib/getConfig.js]: Thank you pointed out. My english is not good.

Comment (yuta0801) [reply lib/getConfig.js]: I fixed the points that were pointed out.

**Comment by yuta0801** [comment]
I fixed the points that were pointed out.

**Comment by Asuza** [comment]
@vinicius0026 I just realized one other thing. I've not used [.nvmrc](https://github.com/fiddus/github-webhooks-listener/blob/master/.nvmrc) before, but probably needs to be updated from `4` to `6`, since template literals are used in this pull request.

Is that file supposed to contain the minimum version of Node that this code runs in?

**Comment by yuta0801** [comment]
Or should not I use template literals?

**Comment by Asuza** [comment]
@yuta0801 I think template literals are fine. Upping the minimum version lets us do some other nice things as well. According to the [nvm readme](https://github.com/creationix/nvm#nvmrc), that's what the file is for. So, I would say to go ahead and change it to a `6`.

**Comment by Asuza** [comment]
@yuta0801 please also update https://github.com/fiddus/github-webhooks-listener/blob/0d41b3ae81c87c77547981e472802d169b6ebba6/package.json#L3 so it's version `0.2.0`, since you added a new feature.

**Comment by vinicius0026** [comment]
I'm in favor of updating node version to the current LTS, that its node >= 8.9.x., that allows us to use es6+ features, such as `async await`.

.nvmrc is for convenience, the correct place to declare the required node version is in the `"engines"` property on `package.json`

**Pull request merged**